dotnet/src/Skills/Skills.OpenAPI/Skills/SkillResourceNames.cs

```diff
@@ -11,4 +11,9 @@ public static class SkillResourceNames
     /// Azure KeyVault skill name.
     /// </summary>
     public const string AzureKeyVault = "AzureKeyVaultSkill";
+
```

Comment:
dont need to modify this file if you use either
kernel.ImportOpenApiSkillFromFile
or kernel.ImportOpenApiSkillFromUrlAsync .
More details below

Reply:
Done